src/AboutUs/View/page.twig

```diff
@@ -1,9 +1,101 @@
-{% include 'Common/View/header.twig' %}
+{# {% include 'Common/View/header.twig' %}
 {% block body %}
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>O nas</h1>
+		<h1 class="u-center-text">Krótki opis naszego zespołu T</h1><br><br><br>
```

Comment:
I think I left some comments to this code already here:https://github.com/womenintechnology/bootcamp-team-t/pull/10